CONTRIBUTING.md

```diff
@@ -333,104 +333,6 @@ via the link above.
 [test-pypi]: https://test.pypi.org/project/pyquil/
 [travis-ci]: https://travis-ci.org/rigetti/pyquil
 
-### Drafting a Release
```

Comment:
These sections didn't seem super relevant anymore, given that Semantic Release will take care of much of it. Please glance over this doc and suggest any more changes (there's a separate story in the pyquil v3 epic for this).